CONTRIBUTING.adoc

```diff
@@ -66,9 +66,9 @@ export OS_OUTPUT_GOPATH=1
 
         $ mkdir -p $GOPATH/src/github.com/openshift
         $ cd $GOPATH/src/github.com/openshift
-        $ git clone git://github.com/<forkid>/origin  # Replace <forkid> with the your github id
+        $ git clone https://github.com/<forkid>/origin  # Replace <forkid> with the your github id
```

Comment:
That might work fine for cloning your repo, but it doesn't work to push changes back to your repo using private key auth. Since this doc is about setting yourself up for contributing, being able to easily push changes to your own repo is essential. What about using the SSH protocol - `git@github.com:<forkid>/origin.git`?